awx/ui/src/screens/Credential/CredentialList/CredentialList.js

```diff
@@ -49,9 +49,8 @@ function CredentialList() {
         CredentialsAPI.readOptions(),
       ]);
       const searchKeys = getSearchableKeys(credActions.data.actions?.GET);
-      const item = searchKeys.indexOf('type');
-      if (item) {
-        searchKeys[item] = 'credential_type__kind';
+      if (credActions.data.actions?.GET.type) {
```

Comment:
When `getSearchableKeys` was introduced we lost the ability to search by credential kind in advanced search.  This reimplements that functionality.